#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_hashbang_fixes.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix missing, weird or otheriwse broken shebang lines

@DPATCH@
diff -urNad hplip~/__init__.py hplip/__init__.py
--- hplip~/__init__.py	2006-03-09 16:27:11.000000000 -0300
+++ hplip/__init__.py	2006-03-16 12:17:32.574316008 -0300
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
 # (c) Copyright 2003-2006 Hewlett-Packard Development Company, L.P.
